oauth2-authorization-server/src/main/java/org/springframework/security/oauth2/server/authorization/client/RegisteredClient.java

```diff
@@ -15,25 +15,17 @@
  */
 package org.springframework.security.oauth2.server.authorization.client;
 
+import org.springframework.lang.Nullable;
+import org.springframework.security.oauth2.core.*;
+import org.springframework.security.oauth2.server.authorization.config.*;
+import org.springframework.util.*;
+
 import java.io.Serializable;
-import java.net.URI;
-import java.net.URISyntaxException;
+import java.net.*;
```

Comment:
Your IDE may be too aggressively condensing imports. Wildcards shouldn't be used.

Reply:
I have removed world card imports